django/db/migrations/serializer.py

```diff
@@ -274,6 +274,8 @@ def serializer_factory(value):
     from django.db.migrations.writer import SettingsReference
     if isinstance(value, Promise):
         value = str(value)
+    elif isinstance(value, SimpleLazyObject):
+        value = value._setupfunc
```

Comment:
I believe is problematic as that would generate something along `MinValueValidator(datetime.datetime.now)` which is broken.

Reply:
Broken in the sense that its incorrect? Because `MinValueValidator(datetime.datetime.now)` will never work? Would a better alternative be: `MinValueValidator(SimpleLazyObject(datetime.datetime.now))`?